app/request_handler.py

```diff
@@ -183,12 +184,21 @@ def _guessYelpId(placeName, lat, lon):
         return cachedId
 
     opts = {
-      'term': placeName[:30],
-      'limit': 1
+      # 'term': placeName,
+      'limit': 20,
+      'radius': 1000,
```

Comment:
Yelp2 APIs use radius_filter, not radius - I can change this before merging though.